src/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation/build/netstandard2.0/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.targets

```diff
@@ -36,7 +36,7 @@
   <Target
     Name="_RunForCore"
     DependsOnTargets="_CreateResponseFileForMvcRazorPrecompile"
-    Condition="'$(TargetFrameworkIdentifier)'=='.NETCoreApp'">
+    Condition="'$(TargetFrameworkIdentifier)'=='.NETCoreApp' AND '$(RuntimeIdentifier)'==''">
```

Comment:
Hmm looks like to be used for restore https://github.com/dotnet/sdk/blob/cd25d6bbfc2aac61cc790c511900427ac64fdaa8/test/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildACrossTargetedLibrary.cs#L129? Regardless, this PR should be fine.